css/portal-dashboard/all-responses-popup/popup-class-nav.less

```diff
@@ -0,0 +1,113 @@
+@import "../variables";
+
+.popupClassNav {
+  width: 240px;
```

Comment:
I think some bold fonts sneaked in.  Add this to a font-weight normal to fix in the popupClassNav class:
`font-weight: normal;`

Reply:
Good catch! I've been assuming all these labels are bold.